src/resource/page/ResourcePage.test.tsx

```diff
@@ -75,6 +95,10 @@ describe(`<ResourcePage />`, () => {
       .spyOn(api, 'getResource')
       .mockImplementation(() => Promise.resolve(testResource));
 
+    jest
+      .spyOn(api, 'getDatePeriod')
+      .mockImplementation(() => Promise.resolve([testDatePeriod]));
+
     const resourcePage = mount(<ResourcePage id={testResource.id} />);
```

Comment:
Actually fixed this in my current pr: the id here is not the same id that our UI applies in URL. Look here: http://hauki-test.oc.hel.ninja/v1/resource/tprek:8215/ According to that the id of that resource is "id": 1186,

Reply:
Ok, I noticed that one so it is the "real" id 😁

Reply:
I'll fix